network/rpc/server/src/bin/server.rs

```diff
@@ -0,0 +1,31 @@
+use nexus_config::{network::rpc::RpcConfig, Config};
+
+use tracing::{level_filters::LevelFilter, Level};
+use tracing_subscriber::{
+    filter, fmt::format::FmtSpan, layer::SubscriberExt, util::SubscriberInitExt,
+};
+
```

Comment:
How reasonable would it be to write an end-to-end test that spins up the client and server, sends a request, and verifies the output?

Reply:
It can be done within a unit test. I don't like the idea of using a real prover for it though.

Reply:
Sounds good, treating this as demo and not production code.

Reply:
I think this one should be kept unresolved until more tests are pushed.

Reply:
Please ping me when these are added and I'll take a look.

Otherwise, I only have non-blocking nitpicks.

Reply:
The new tests at `network/rpc/server/src/tests/mod.rs` address this thread.